apps/sms/views/conversation/style/composer.css

```diff
@@ -4,12 +4,6 @@
   background: #e1f0ec;
```

Comment:
What do you think if we clean Composer component style file a little bit more?

* ```#composer-container``` to ```coversation.css``` or ```composer_view.css```;
* ```.flex-composer``` to ```coversation.css``` or ```composer_view.css```;
* ```#messages-container ``` to ```coversation.css```;
* ```section[role="region"].new > gaia-header h1``` to ```composer_view.css```;
* ```#messages-contact-pick-button``` to ```coversation.css``` or ```composer_view.css```;
* ```.subheader menu``` to ```composer_view.css```;
* ```.subheader .overlay``` to ```composer_view.css```;